src/client/pages/create.js

```diff
@@ -225,11 +242,18 @@ class Create extends React.Component {
                 <td>
                   <a href={`${window.location.origin}/${this.state.gameID}/${i}/${this.state.secret[i]}`} target="_blank" rel="noopener noreferrer">{window.location.origin}/{this.state.gameID}/{i}/{this.state.secret[i]}</a>
                 </td>
+                <td>
+                  <Button onClick={() => this.copyToClipboard(`${window.location.origin}/${this.state.gameID}/${i}/${this.state.secret[i]}`)}>
```

Comment:
As this URL is created at 3 different places, could you refactor it to a function `i => url(i)` ?

Reply:
Idea:  Would it look nicer if we didn't use a button but rather an icon like https://materialui.co/icon/content-copy ?

Reply:
And yet another idea: I think on other websites there usually is an animation when copying to the clipboard (like an appearing check mark or so). I believe from a UX perspective this is done to give feedback to the user that something did indeed happen.  Can we include this easily?

Reply:
I've refactored the url generation into a function.

I agree that an icon might look a bit neater and the tick's probably a good idea to say whether copying was successful